### DISCUSSION TITLE
Clear text password field and storage

### DISCUSSION THREAD

**Discussion by 6wheels** [Koenkk/zigbee2mqtt#14364] [General]
Hi,

I just added an auth-token in the configuration, and I was surprised that the password window is in clear text as well as the password value in the local storage.

Is it planned to improve this?

**Comment by Koenkk** [comment]
Please create issues related to the frontend here: https://github.com/nurikk/zigbee2mqtt-frontend/issues

**Reply by 6wheels** [reply]
Got it! Thanks!
